genesyscloud/resource_genesyscloud_telephony_providers_edges_edge_group.go

```diff
@@ -0,0 +1,230 @@
+package genesyscloud
+
+import (
+	"context"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/mypurecloud/platform-client-sdk-go/v48/platformclientv2"
+	"log"
+)
+
+func resourceEdgeGroup() *schema.Resource {
+	return &schema.Resource{
+		Description: "Genesys Cloud Edge Group",
+
+		CreateContext: createWithPooledClient(createEdgeGroup),
+		ReadContext:   readWithPooledClient(readEdgeGroup),
+		UpdateContext: updateWithPooledClient(updateEdgeGroup),
+		DeleteContext: deleteWithPooledClient(deleteEdgeGroup),
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		SchemaVersion: 1,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Description: "The name of the entity.",
+				Type:        schema.TypeString,
+				Required:    true,
+			},
+			"description": {
+				Description: "The resource's description.",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"state": {
+				Description: "Indicates if the resource is active, inactive, or deleted.",
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+			},
+			"managed": {
+				Description: "Is this edge group being managed remotely.",
+				Type:        schema.TypeBool,
+				Optional:    true,
+			},
+			"hybrid": {
+				Description: "Is this edge group hybrid.",
+				Type:        schema.TypeBool,
+				Optional:    true,
```

Comment:
Since this is optional, should it have a default of false?

Reply:
Yeah sounds good